specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -456,7 +456,7 @@
           "type": "string",
           "description": "the id of the action group to use."
         },
-        "webhookProperties": {
+        "webHookProperties": {
```

Comment:
Changing a property name in stable API version is breaking change. Suggest to change it in new API version.

Reply:
@raych1 
Thanks for the review.

It is a bug fix. There is a misalignment between metric alerts RP and metric alerts swagger.
Currently Metric alerts RP returns webHookProperties while swagger documented webhookProperties.

Because of this misalignment, azure_cli returns both webHookProperties and webhookProperties. (Open issue for broken CLI: https://github.com/Azure/azure-cli/issues/10106)

As the RP already returns the new property name (instead of the existing one), can we push it to the current API version?
Any other suggestion how to fix it?

Reply:
It's ok to align with the RP in this case.